### PR TITLE
Add AI for Game Dev part 3 post

### PR DESCRIPTION
**Pull request by dylanebert**

### CODE REVIEW

**Review by simoninithomas — APPROVED** [review]
Very good article, I like the overview of the different technologies and the thumbnail.
The only thing I would add (optional) but maybe talk about the different services that exists such as scenario.gg etc? WDYT?

File: ml-for-games-3.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+</figure>
+
+Things are changing rapidly in this area, though, and there may be a viable solution in the near future. Next, I'll talk about some of the directions text-to-3D may be going.
+
+### The Future of Text-to-3D
+
+While text-to-3D has come a long way recently, there is still a significant gap between where we are now and what could have an impact along the lines of text-to-image. I can only speculate on how this gap will be closed. There are two possible directions that are most apparent:
+
+1. Improvements in NeRF-to-mesh and mesh generation. Currently, NeRF-to-mesh and direct mesh generation models like [CLIPMatrix](https://arxiv.org/abs/2109.12922) result in meshes that require significant work and expertise to become game-ready assets, similar to photogrammetry, which has been around for a while. While this is useful in some scenarios, like creating realistic high-fidelity assets, it's still more time-consuming than making low-poly assets from scratch, especially if you're like me and use an ultra-low-poly art style.
```
Comment (pcuenca) [inline ml-for-games-3.md:86]: The first sentence reads very similar to me to what was discussed a few paragraphs above. Maybe we could rewrite it slightly to make it sound less redundant?

```suggestion
1. Improvements in NeRF-to-mesh and mesh generation. As we've seen, current generation models are similar to photogrammetry in that they require a lot of work to produce game-ready assets. While this is useful in some scenarios, like creating realistic high-fidelity assets, it's still more time-consuming than making low-poly assets from scratch, especially if you're like me and use an ultra-low-poly art style.
```

**Review by pcuenca — APPROVED** [review]
Nice post! I really liked that you provide pointers at the beginning of each section to the relevant techniques and "pre-requisites". Just made a couple of very minor suggestions.

**Comment by dylanebert** [comment]
> Very good article, I like the overview of the different technologies and the thumbnail. The only thing I would add (optional) but maybe talk about the different services that exists such as scenario.gg etc? WDYT?

Since it doesn't generate actual 3D assets, just images, maybe I can include it in the next part, on 2D assets?

**Pull request merged**